app/services/configuration_repository_service.rb

```diff
@@ -57,20 +65,71 @@ def configuration_repository_valid?
       return valid
     end
 
+    # Adds the bot user as a collaborator to the fastlane.ci configuration
+    # repository
+    #
+    # @return [Boolean] If the user was added successfully
+    def add_bot_user_as_collaborator
+      invitation_id = invite_bot_user_to_configuration_repository
+      return accept_invitation_to_repository_as_bot_user(invitation_id) if invitation_id
```

Comment:
what happens if we don't have an `invitation_id`?

Reply:
It will error out [here](https://github.com/fastlane/ci/pull/661/files/791d9f1453f6cc7301e1377ae467d3dd92d2d965#diff-7f1f37a512473c728fda7e56977fbdeeR126).

Would you like me to do error handling in this method as well?

Reply:
Yeah but if you get to line `74`, and there's no `invitation_id` it seems like `add_bot_user_as_collaborator` just fails silently. Only getting a log message. Maybe we should crash?